docs/data_model.md

```diff
@@ -199,92 +196,4 @@ perspective.
-role tokens.
-
-A principal token is serialized as a string with following attributes:
-
--   version (v) - the version of the token - U1 for user tokens and S1
-    for service tokens
--   domain (d) - the domain of principal. For users this will always be
-    "user"
--   name (n) - the name of the principal
--   host (h) - the FQDN of the host that issued this token
--   salt (a) - a salt
--   time (t) - the time the token was issued
--   expires (e) - the time the token expires
--   keyId (k) - the key identifier of the private key that was used to
-    sign this token
--   ip (i) - the IP address where the request to get this token was
-    initiated from
--   authorized-services (b) - the list of services authorized to use
-    this user token
--   signature (s) - the signature of the other items
-
-The single letter in parentheses is the key in the serialized string,
-where semicolons separate key value pairs, and the key and value are
-separated by an equals sign.
-
-For example:
-
-    v=U1;d=user;n=joe;h=host.athenz.org;a=bb4e;t=1442255237;e=1442258837;k=0;i=10.72.42.32;s=Jw8SvYGYrk
-
-Note that requests that include a principal token should be protected to
-keep the token from being intercepted and reused (for the lifetime of
-the token). It is strongly recommended to always use HTTPS for such
-requests.
-
-### Role Token (ZToken)
------------------------
-
-Role tokens represent an authoritative statement that a given principal
-may assume some number of roles in a domain for a limited period of
-time. Like NTokens, they are signed to prevent tampering. In a sense,
-the ZTS is an authority except that it is an authority
-over Roles instead of Users.
-
-A role token is serialized as a string with following attributes:
-
--   version (v) - the version of the token - Z1
--   domain (d) - the domain of role token
--   roles (r) - a list of comma-separated role names
--   principal (p) - the name of the principal (user/service) who
-    requested this role token
--   host (h) - the FQDN of the ZTS host that issued this token
--   salt (a) - a salt
--   time (t) - the time the token was issued
--   expires (e) - the time the token expires
--   keyId (k) - the key identifier of the ZTS private key that was
-    used to sign this token
--   ip (i) - the IP address where the request to get this token was
-    initiated from
--   signature (s) - the signature of the other items
-
-The single letter in parentheses is the key in the serialized string,
-where semicolons separate key value pairs, and the key and value are
-separated by an equals sign.
-
-For example:
-
-    v=Z1;d=media.news;r=editor;p=user.joe;h=host.athenz.com;a=bb4e;t=1442255237;e=1442258837;k=0;i=10.72.42.32;s=P43Vp_LQh1"
-
-Note that requests that include a ZToken should be encrypted to keep the
-ZToken from being intercepted and reused (for the lifetime of the
-token). It is strongly recommended to always use HTTPS for such
-requests.
-
```

Comment:
We should mention Access tokens here with appropriate generic details about claims in the token.

Reply:
OK. I also just noticed that I searched for "N Token" and not "NToken" so I missed quite a few references.
I'll push a fix to the PR by tomorrow

Reply:
Removed everything except the references in the authorization flows (centralized / decentralized and the examples). Will replace them on another PR.